src/BinaryParsers/PEBinary/ProgramDatabase/ObjectModuleDetails.cs

```diff
@@ -224,28 +224,35 @@ public string GetDialect(out string versionNumber)
         }
     }
 
+    /// <summary>
+    /// Information about the CV_CFL_LANG enumeration, 
+    /// which specifies the code language of the application or linked module in the debug interface access SDK.
+    /// https://docs.microsoft.com/en-us/visualstudio/debugger/debug-interface-access/cv-cfl-lang
+    /// </summary>
     public enum Language : uint
     {
         C = 0x00,
         Cxx = 0x01,
         FORTRAN = 0x02,
-        MASM = 0x03,
+        MASM = 0x03, // Microsoft Macro Assembler
         Pascal = 0x04,
         Basic = 0x05,
         COBOL = 0x06,
-        LINK = 0x07,
-        CVTRES = 0x08,
-        CVTPGD = 0x09,
+        LINK = 0x07, // linker-generated module
+        CVTRES = 0x08, // resource module converted with CVTRES tool
+        CVTPGD = 0x09, // POGO optimized module generated with CVTPGD tool
         CSharp = 0x0A,
         VisualBasic = 0x0B,
-        ILASM = 0x0C,
+        ILASM = 0x0C, // intermediate language assembly (that is, Common Language Runtime (CLR) assembly)
         Java = 0x0D,
         JScript = 0x0E,
-        MSIL = 0x0F,
+        MSIL = 0x0F, // Microsoft Intermediate Language (MSIL), possibly a result of using the /LTCG (Link-time Code Generation) switch
```

Comment:
>LTCG [](http://example.com/codeflow?start=96&length=4)

Huh? The native whole code optimization feature produces MSIL? Can you show me the source of your information?

Reply:
[https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/debugger/debug-interface-access/cv-cfl-lang.md](https://github.com/MicrosoftDocs/visualstudio-docs/blob/main/docs/debugger/debug-interface-access/cv-cfl-lang.md)